components/sensor/pulse_meter.rst

```diff
@@ -6,8 +6,16 @@ Pulse Meter Sensor
     :image: pulse.svg
 
 The pulse meter sensor allows you to count the number and frequency of pulses on any pin. It is intended to be a drop-in replacement
-for :doc:`integration sensor </components/sensor/pulse_counter>`, but offering better resolution.
-It measures the time between rising edges on a pin, for each pulse it outputs the frequency in pulses/min.
+for the :doc:`pulse counter integration </components/sensor/pulse_counter>`.
+It measures the time between rising edges on a pin, for each pulse it outputs the frequency in ``pulses/min``, as opposed to the pulse counter sensor which outputs the number of pulses in a time interval.
```

Comment:
```suggestion
It measures the _width_ of pulses on a pin, and converts this to an equivalent frequency expressed in pulses per minute, as opposed to the pulse counter sensor which counts the number of pulses in a time interval.
```

Reply:
But it doesn't measure the width of the pulses. It measures the time between two rising edges, at least according to the documentation that was in place before. Like so:

![images](https://github.com/esphome/esphome-docs/assets/29854606/dae924b6-3f48-4e8e-b2dd-dca6df8b6b1b)

So *not* the width of the pulses.

Reply:
It 100% does not measure the width of the pulses, it always measures rising edge to rising edge